### ISSUE TITLE
[verify_xcode] Cannot verify Xcode installed from https://developer.apple.com/downloads/ 

### ISSUE THREAD

**Issue by gbegen** [fastlane/fastlane#4073]
### Issue Description
When Xcode was installed from Apple Downloads instead of the Mac App Store, the verify_xcode action fails.

##### Complete output when running fastlane, including the stack trace and command used

```
$ fastlane run verify_xcode
[12:42:27]: --------------------------
[12:42:27]: --- Step: verify_xcode ---
[12:42:27]: --------------------------
[12:42:27]: Verifying your Xcode installation at path '/Applications/Xcode-7.2.1.app'...
[12:42:27]: Verifying Xcode was signed by Apple Inc.
[12:42:27]: $ codesign --display --verbose=4 '/Applications/Xcode-7.2.1.app'
[12:42:29]: ▸ Executable=/Applications/Xcode-7.2.1.app/Contents/MacOS/Xcode
[12:42:29]: ▸ Identifier=com.apple.dt.Xcode
[12:42:29]: ▸ Format=bundle with Mach-O thin (x86_64)
[12:42:29]: ▸ CodeDirectory v=20100 size=307 flags=0x0(none) hashes=7+5 location=embedded
[12:42:29]: ▸ Hash type=sha1 size=20
[12:42:29]: ▸ CDHash=dc74c741271f69129ef4121960857bdd171f19cd
[12:42:29]: ▸ Signature size=4105
[12:42:29]: ▸ Authority=Software Signing
[12:42:29]: ▸ Authority=Apple Code Signing Certification Authority
[12:42:29]: ▸ Authority=Apple Root CA
[12:42:29]: ▸ Info.plist entries=34
[12:42:29]: ▸ TeamIdentifier=not set
[12:42:29]: ▸ Sealed Resources version=2 rules=12 files=387668
[12:42:29]: ▸ Internal requirements count=1 size=68
[12:42:29]: Attention: Your Xcode Installation might be hacked.
[12:42:29]: This might be a false alarm, if so, please submit an issue on GitHub
[12:42:29]: The following information couldn't be found:
[12:42:29]: Authority=Apple Mac OS Application Signing
Authority=Apple Worldwide Developer Relations Certification Authority
TeamIdentifier=59GAB85EFG

[!] The Xcode installation at path '/Applications/Xcode-7.2.1.app' might be compromised.
```

### Environment

fastlane version (fastlane -v): 1.80.0
Do you use bundler, rbenv or rvm? No

### Analysis

This appears to be because Xcode versions available from Apple downloads at https://developer.apple.com/downloads/ are signed differently that those from the Mac App Store.

#### From Mac App Store

```
$ codesign --display --verbose=4 /Applications/Xcode.app
Executable=/Applications/Xcode.app/Contents/MacOS/Xcode
Identifier=com.apple.dt.Xcode
Format=bundle with Mach-O thin (x86_64)
CodeDirectory v=20200 size=322 flags=0x200(kill) hashes=7+5 location=embedded
Hash type=sha1 size=20
CDHash=81b3558eb642155224b84848faa2ba69def86c6d
Signature size=4167
Authority=Apple Mac OS Application Signing
Authority=Apple Worldwide Developer Relations Certification Authority
Authority=Apple Root CA
Info.plist entries=34
TeamIdentifier=59GAB85EFG
Sealed Resources version=2 rules=12 files=387668
Internal requirements count=1 size=108
```

#### From Apple Downloads

```
$ codesign --display --verbose=4 /Applications/Xcode-7.2.1.app
Executable=/Applications/Xcode-7.2.1.app/Contents/MacOS/Xcode
Identifier=com.apple.dt.Xcode
Format=bundle with Mach-O thin (x86_64)
CodeDirectory v=20100 size=307 flags=0x0(none) hashes=7+5 location=embedded
Hash type=sha1 size=20
CDHash=dc74c741271f69129ef4121960857bdd171f19cd
Signature size=4105
Authority=Software Signing
Authority=Apple Code Signing Certification Authority
Authority=Apple Root CA
Info.plist entries=34
TeamIdentifier=not set
Sealed Resources version=2 rules=12 files=387668
Internal requirements count=1 size=68
```

Perhaps verify_xcode should permit both variations of Authority and TeamIdentifier entries.

**Comment by TKBurner** [comment]
Thanks for reaching out! I'm sorry to hear that this is happening. The team here is going to have a look. Thanks!

**Comment by KrauseFx** [comment]
Seems to be related to https://github.com/fastlane/fastlane/issues/661

**Comment by gbegen** [comment]
Yeah, seems related to where the discussion on #661 went, but is not the error originally reported as #661.

**Comment by mfurtak** [comment]
Reproduced locally. :+1: I'm going to adjust the code to accept the information from the Apple Downloads binary as well.

**Comment by mfurtak** [comment]
Hi @gbegen - I have merged #4159 to address this, so it should go out in the next release! :rocket:

**Comment by TKBurner** [comment]
@gbegen I'm going to close this issue out. Update us after the fix goes live if you still are running into any issues 👍
